### ISSUE TITLE
need help with recovering a vod

### ISSUE THREAD

**Issue by Frauu2** [TwitchRecover/TwitchRecover#224]
Hello,as I said above, I need help recovering this vod when entering the time, the launcher closes. In the links it appears that the live has started at two different hours of start, I have used both and the same thing happens.Thanks in advance

https://streamscharts.com/channels/saramartinng/streams/39383896968
https://twitchtracker.com/saramartinng/streams/39383896968

**Comment by ItIckeYd** [comment]
@Frauu2 You can use the following python script to recover vod.

https://github.com/ItIckeYd/VodRecovery

this repository is no longer being updated and takes time to troubleshoot the issue

**Comment by Frauu2** [comment]
Hey @ItIckeYd i have already try It to use but i don't if i am dumb but i couldn't make It work

**Comment by ItIckeYd** [comment]
@Frauu2 make sure you are using local timezone. 

Example: Korea is GMT+9 which is UTC+9 which means if twitchtracker is giving a start time in UTC… you need to add 9 hours to it.

Make sure you are using vod id from twitchtracker and the vod is not older then 60 days. 

if it still doesn't work give me an example of what you tried?

**Comment by Redfly020** [comment]
hello my links doesnt work can u help me https://dgeft87wbj63p.cloudfront.net/5f135d927a0a77a3dbab_evalunna_45150274844_1649163578/chunked/index-dvr.m3u8 https://twitchtracker.com/evalunna/streams/45150274844

**Comment by ItIckeYd** [comment]
@Redfly020 did you check segments? If you have the m3u8 link the script should say exactly the reason it’s not working (only reason I know of so far is segments).

**Comment by Redfly020** [comment]
what is segments ? where i found segments

**Comment by ItIckeYd** [comment]
@Redfly020 if you have the latest version of the script it will ask you if you want to check the segments… depending on the results of that it should tell you if the vod is playable

**Comment by Redfly020** [comment]
ok thx last question i have to unmute the vod or no ?

**Comment by ItIckeYd** [comment]
@Redfly020 scripts tell you that information as well.

**Comment by Redfly020** [comment]
the script give me this M3U8 DOES NOT contain any muted segments

**Comment by ItIckeYd** [comment]
@Redfly020 right.. given it does not contain muted segments there is no reason to unmute it correct?

**Comment by Redfly020** [comment]
so what i have to do im lost

**Comment by Soapzz** [comment]
@ItIckeYd Hi, I can't seam to make the script you gave us work, could you help me and see if you can recover this VOD please ?
https://streamscharts.com/channels/evalunna/streams/45435758364
I would really appreciate it

**Comment by ItIckeYd** [comment]
@Soapzz 

https://dgeft87wbj63p.cloudfront.net/de5645d48738ccc88983_evalunna_45435758364_1653052705/chunked/index-dvr.m3u8
https://d1ymi26ma8va5x.cloudfront.net/de5645d48738ccc88983_evalunna_45435758364_1653052705/chunked/index-dvr.m3u8
Vod does NOT contain muted segments

**Comment by Soapzz** [comment]
@ItIckeYd Wow, thank you so so much, here :
https://twitter.com/Soapz__
Please DM me your Paypal, I really want to give you something, I'd never gotten the VOD without your help, oh and can you tell me your Twitter @ here so I'm sure it's you when I get the DM (I will delete this message after so nobody else has my Twitter, I suggest you do the same)

**Comment by ItIckeYd** [comment]
@Soapzz I don't currently have twitter. I also will not accept any money or anything like that.. Please keep it and save up for something that you need... the world is getting more and more expensive as time goes on.

**Comment by Soapzz** [comment]
@ItIckeYd Alright then, I respect it, again, thank you very much for your help, I won't forget it, have a great one !

**Comment by thegamingstig** [comment]
Hi there,

Sorry to add in, but see if you can find the VOD for this. I tried the revised twitchrecover and vodrecovery but no success.

https://twitchtracker.com/sukiyuki3/streams/46450999357

**Comment by ItIckeYd** [comment]
@thegamingstig Doesn't exist.. must have been removed right away.
